CMakeLists.txt

```diff
@@ -29,11 +31,11 @@ include(cmake/Dependencies.cmake)
 
 # ---[ Flags
 if(UNIX OR APPLE)
-  set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -fPIC -Wall")
+  set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -fPIC -Wall -std=c++11")
```

Comment:
Ty for the big contribution :+1: .

Would -std=c++0x work? I think travis runs GCC 4.6 and only GCC 4.7 and later support -std=c++11.